ChangeLog

```diff
@@ -30,6 +30,7 @@ ver. 0.9.5 (2016/XX/XXX) - wanna-be-released
    * New Actions:
      - action.d/firewallcmd-rich-rules and action.d/firewallcmd-rich-logging (gh-1367)
 - Enhancements:
+   * extremely speedup of all sqlite database operations (gh-1436)
```

Comment:
> Extreme ... by using temp_store memory

There are 3 pragmas in this FR, and temp_store currenly increase speed of upgrade only
A reference to PR is in the same line (gh-1436) where described all this, otherwise there is `git blame`, that can show all the changes;

> Shouldn't it be an option in fail2ban.conf so if it doesn't work for them they could quickly disable it?

:+1: 
I'll do that in 0.10 together with any other options, that I promised (and then cherry pick back this options to 0.9)